noark-extraction-validator/src/main/resources/noark53/noark53-validation.xml

```diff
@@ -356,58 +370,556 @@
+
+                    UNION ALL
+
+                    SELECT 'total', count(*), TRUE in_document_description, TRUE in_record
+                    FROM arkivstruktur.dokumentobjekt;
+                ]]>
+            </info>
+        </queries>
+    </check>
+
+    <check id="ASC13">
+        <title>Case Party count</title>
+        <description>Number of Case Parties grouped by Series</description>
+        <group>arkivstruktur</group>
+        <queries>
+            <info>
+                <![CDATA[
+                    SELECT f._ref_arkivdel series_system_id, COUNT(*) number_of_case_parties
+                    FROM arkivstruktur.sakspart cp
+                    JOIN arkivstruktur.mappe f ON f.systemid = cp._ref_mappe
+                    GROUP BY f._ref_arkivdel;
+                ]]>
+            </info>
+        </queries>
+    </check>
+
+    <check id="ASC14">
+        <title>Note count</title>
+        <description>Number of Notes related to Files, Records, and Document Descriptions grouped by Series</description>
+        <group>arkivstruktur</group>
+        <queries>
+            <info>
+                <![CDATA[
+                    SELECT s.systemid series_system_id,
+                        COALESCE(f.val, 0) number_of_file_notes,
+                        COALESCE(r.val, 0) number_of_record_notes,
+                        COALESCE(d.val, 0) number_of_document_notes
+                    FROM arkivstruktur.arkivdel s
+                    LEFT JOIN
+                    (
+                        SELECT f._ref_arkivdel series_system_id, COUNT(*) val
+                        FROM arkivstruktur.merknad n
+                        JOIN arkivstruktur.mappe f ON f.systemid = n._ref_mappe
+                        GROUP BY f._ref_arkivdel
+                    ) f ON f.series_system_id = s.systemid
+                    LEFT JOIN
+                    (
+                        SELECT r._ref_arkivdel series_system_id, COUNT(*) val
+                        FROM arkivstruktur.merknad n
+                        JOIN arkivstruktur.registrering r ON r.systemid = n._ref_registrering
+                        GROUP BY r._ref_arkivdel
+                    ) r ON r.series_system_id = s.systemid
+                    LEFT JOIN
+                    (
+                        SELECT r._ref_arkivdel series_system_id, COUNT(*) val
+                        FROM arkivstruktur.merknad n
+                        JOIN arkivstruktur.dokumentbeskrivelse d ON d.systemid = n._ref_dokumentbeskrivelse
+                        JOIN arkivstruktur.registrering r ON r.systemid = d._ref_registrering
+                        GROUP BY r._ref_arkivdel
+                    ) d ON d.series_system_id = s.systemid;
+                ]]>
+            </info>
+        </queries>
+    </check>
+
+    <check id="ASC15">
+        <title>Cross reference count</title>
+        <description>Number of cross references related to Classes, Files, and Records grouped by Series</description>
```

Comment:
`Number of cross references associated with classes, files, or records, grouped by series`